algebird-core/src/main/scala/com/twitter/algebird/DecayedVector.scala

```diff
@@ -51,13 +53,16 @@ object DecayedVector extends CompatDecayedVector {
 
   def mapMonoidWithEpsilon[K](
       eps: Double
-  )(implicit vs: VectorSpace[Double, ({ type x[a] = Map[K, a] })#x], metric: Metric[Map[K, Double]]) =
+  )(implicit
+      vs: VectorSpace[Double, ({ type x[a] = Map[K, a] })#x],
+      metric: Metric[Map[K, Double]]
+  ): Monoid[DecayedVector[({ type x[a] = Map[K, a] })#x]] =
     monoidWithEpsilon[({ type x[a] = Map[K, a] })#x](eps)
 
   implicit def mapMonoid[K](implicit
       vs: VectorSpace[Double, ({ type x[a] = Map[K, a] })#x],
       metric: Metric[Map[K, Double]]
-  ) =
+  ): Monoid[DecayedVector[({ type x[a] = Map[K, a] })#x]] =
```

Comment:
One follow up PR maybe we can bring kind-projector to make these a little nice.

Reply:
be happy to, especially now that the syntax is set for scala 3